tests/lighthouse/lighthouserc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ci": {
+    "collect": {
+      "upload": {
+        "target": "filesystem",
+        "outputDir": "./tests/lighthouse/reports/"
+      },
+      "settings": {
+        "chromeFlags": "--headless"
+      }
+    },
+    "assert": {
+      "assertions": {
+        "categories:performance": ["warn", { "minScore": 0.9 }],
```

Comment:
these all need to be errors, not warnings. Errors will force the script to return a failure code and therefore fail the step in pipelines.